plugins/woocommerce/client/blocks/docs/internal-developers/testing/releases/761.md

```diff
@@ -15,7 +15,8 @@ Zip file for testing: [woocommerce-gutenberg-products-block.zip](https://github.
 | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
 | ![Screenshot showing a PHP warning](https://user-images.githubusercontent.com/3616980/172639823-1b671ac0-58ea-4f4b-9b00-aa481d4acd72.png) | ![Screenshot showing no PHP warning](https://user-images.githubusercontent.com/3616980/172639911-7a1416fe-f49d-4c58-90b7-7e07637b1777.png) |
 
-### Replace instances of wp_is_block_theme() with wc_current_theme_is_fse_theme(). ([6590](https://github.com/woocommerce/woocommerce-blocks/pull/6590))
+### Replace instances of wp_is_block_theme() with wp_is_block_theme()
```

Comment:
It looks like you've done a blanket find/replace, but we don't want these docs to be edited.